### PR TITLE
Updating with merged nodejs imagestream with RHOAR from RHSCL

### PR DESCRIPTION
**Pull request by sspeiche**
@lance FYI

### CODE REVIEW

File: official.yaml [hunk 1]
```diff
@@ -52,8 +52,8 @@ data:
           - online-starter
           - online-professional
     imagestreams:
-      - location: https://raw.githubusercontent.com/sclorg/s2i-nodejs-container/master/imagestreams/nodejs-rhel7.json
-        docs: https://github.com/sclorg/s2i-nodejs-container/blob/master/README.md
+      - location: https://raw.githubusercontent.com/bucharest-gold/centos7-s2i-nodejs/master/imagestreams/nodejs-rhel7.json
+        docs: https://github.com/sbucharest-gold/centos7-s2i-nodejs/blob/master/README.md
```
Comment (bparees) [inline official.yaml]: "sbucharest"?

**Comment by bparees** [comment]
fix what I assume is a typo and lgtm.

I believe @coreydaley has merge permissions on this repo if i'm not available.

File: official/nodejs/templates/nodejs-example.json [hunk 2]
```diff
@@ -105,7 +105,7 @@
                         ],
                         "from": {
                             "kind": "ImageStreamTag",
-                            "name": "nodejs:6",
+                            "name": "nodejs:${NODEJS_VERSION}",
```
Comment (sspeiche) [inline official/nodejs/templates/nodejs-example.json]: @bparees Hmm, I didn't update my branch to include your updates but I guess import/verify did this? Not sure what I need to do to correct this.  Perhaps @coreydaley has thoughts

Comment (bparees) [reply official/nodejs/templates/nodejs-example.json]: make import pulls all updated content.  looks like my PR (https://github.com/sclorg/nodejs-ex/pull/177) merged and you picked it up on your import.  it's fine.

Comment (bparees) [reply official/nodejs/templates/nodejs-example.json]: (I did that PR so people can actually choose to use the rhoar imagestreamtag when deploying the template if they want.  I though you'd be happy :) )

**Comment by bparees** [comment]
lgtm. will merge when travis agrees.

**Pull request merged**